src/odoc/pages.mld

```diff
@@ -0,0 +1,68 @@
+{0 Documentation pages}
```

Comment:
Any chance we can elaborate on the current pages? We have a [`using-odoc` page](http://ocaml.github.io/odoc/odoc/using-odoc.html#custom-pages) that is intended to evolve and describe the features `odoc` includes.